vnext/Microsoft.ReactNative/IReactContext.idl

```diff
@@ -2,13 +2,20 @@
 // Licensed under the MIT License.
 
 import "IJSValueWriter.idl";
+import "ReactPropertyBag.idl";
 
 #include "NamespaceRedirect.h"
 
 namespace Microsoft.ReactNative {
 
   [webhosthidden]
   interface IReactContext {
+    // A singleton property bag global for the whole application.
```

Comment:
>application [](start = 53, length = 11)

Nit, for my own education: Is it correct to assume that this is not related to "Xaml application" but more fundamental, like "process"?

Reply:
Yes, this is global to the whole process. Should I re-phrase the comment?

Reply:
The global properties are removed after an offline discussion with @acoates-ms .